x-pack/legacy/plugins/apm/public/components/shared/Summary/index.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiFlexGrid, EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import { px, units } from '../../../../public/style/variables';
+
+interface Props {
+  items: Array<React.ReactElement | null | undefined>;
```

Comment:
Why both `null` and `undefined`? Can't we pick one?

Reply:
We can, but that puts the onus on the consumer of this component to make sure that any falsey value is either null or undefined, which is not the case right now because we use `{condition} && {component}` in some places, rather than `{condition} ? {component} : null`.

Reply:
(another great use case for the null-coalescing operator)

Reply:
Okay, good point. Leave it be.